release-to-candidate/README.md

```diff
@@ -30,6 +30,7 @@ jobs:
 | `channel`                | The channel to release the snap to.                                                       |    N     | `latest/candidate` |
 | `launchpad-token`        | A token with permissions to create Launchpad remote builds.                               |    Y     |                    |
 | `repo-token`             | A token with privileges to create and push tags to the repository.                        |    Y     |
+| `show-build-log`         | Show the build log even if the build succeed                                              |    N     | `'true'`           |
```

Comment:
Let's drop this completely, and squash this PR down into one commit that just makes the `disable-fallback` environment change -- the `git init` change shouldn't be required, as all of these workflows are triggered from git repos anyway

Reply:
`git init` is needed where custom project root  is used eg the ffmpeg one. Because snapcraft cannot detect git from the parent directory.

Reply:
Okay, let's leave the `git init` part in, but I think we should remove the `show-build-log` stuff and then we can merge.